src/main/java/forestry/core/utils/Translator.java

```diff
@@ -2,25 +2,24 @@
 
 import java.util.IllegalFormatException;
 
-import net.minecraft.util.text.translation.I18n;
+import net.minecraft.client.resources.I18n;
 
 public class Translator {
 	private Translator() {
 
 	}
 
-	@SuppressWarnings("deprecation")
 	public static String translateToLocal(String key) {
-		if (I18n.canTranslate(key)) {
-			return I18n.translateToLocal(key);
+		if (I18n.hasKey(key)) {
+			return I18n.format(key);
 		} else {
-			return I18n.translateToFallback(key);
+			return net.minecraft.util.text.translation.I18n.translateToFallback(key);    //no equivalent from what I can see
```

Comment:
This doesn't seem to have a replacement. If we don't use it the unformatted String will be displayed. Not sure what the best option is.